src/tools/run-make-support/src/lib.rs

```diff
@@ -1,16 +1,28 @@
 use std::env;
 use std::path::{Path, PathBuf};
 use std::process::{Command, Output};
-
+mod rustdoc;
+pub use rustdoc::rustdoc;
 pub use wasmparser;
 
+pub fn add_host_rpath_env(cmd: &mut Command) {
+    let ld_lib_path_envvar = env::var("LD_LIB_PATH_ENVVAR").unwrap();
+    let ld_lib_path_value = env::var(&ld_lib_path_envvar).unwrap();
+
+    let temp = env::var("TMPDIR").unwrap();
+    let host_rpath_dir = env::var("HOST_RPATH_DIR").unwrap();
+
+    cmd.env(ld_lib_path_envvar, format!("{temp}:{host_rpath_dir}:{ld_lib_path_value}"));
+}
+
 pub fn out_dir() -> PathBuf {
     env::var_os("TMPDIR").unwrap().into()
```

Comment:
Should we be using https://doc.rust-lang.org/nightly/std/env/fn.temp_dir.html?

Reply:
(this TMPDIR is some directory under `build/`, not `tmp/`)